### PR TITLE
{bio}[foss/2019b] BUSCO v4.0.5, AUGUSTUS v3.3.3, SEPP v4.3.10, ... w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

edit: requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1998~~

### CODE REVIEW

File: easybuild/easyconfigs/s/SEPP/SEPP-4.3.10-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
```
Comment (smoors) [inline easybuild/easyconfigs/s/SEPP/SEPP-4.3.10-foss-2019b.eb]:
```suggestion

easyblock = 'PythonPackage'

```

Comment (boegel) [reply easybuild/easyconfigs/s/SEPP/SEPP-4.3.10-foss-2019b.eb:3]: No, this requires https://github.com/easybuilders/easybuild-easyblocks/pull/1998

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0459ca81ce41c88107aa8c3c49fc20b9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2646.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/455bcfedda2452425d03ee88d58c16c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/99c4b9fc04b590f0ead68f075bd221d3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3113.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ba41f171a1e3b0fc191decb1b0bdbc4c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**